Source/CarthageKitTests/ProjectSpec.swift

```diff
@@ -162,6 +165,14 @@ class ProjectSpec: QuickSpec {
 
 				assertProjectEvent(commitish: commitish) { expect($0).to(beNil()) }
 			}
+
+			it ("should not fetch twice in a row, even if no commitish is given") {
```

Comment:
it's the previous git parlance used and the name of the parameter into the clone/fetch methods. Basically it can be a commit sha or some other 'commit-ish' ref like a tag or branch